resources/js/common/components/AchievementsListItem/AchievementsListItem.tsx

```diff
@@ -71,19 +70,29 @@ export const AchievementsListItem: FC<AchievementsListItemProps> = ({
           <div className="mb-0.5 flex justify-between gap-x-2">
             {/* Title */}
             <div className="-mt-2 mb-0.5 md:mt-0">
-              <a href={route('achievement.show', { achievement })} className="font-medium">
-                {title}
-                {game?.title ? ' ' : null}
-              </a>
-
-              {game?.title ? (
-                <Trans
-                  i18nKey="<1>from</1> <2>{{gameTitle}}</2>"
-                  components={{
-                    1: <span />,
-                    2: <AchievementGameTitle game={game} />,
-                  }}
-                />
+              <span className="mr-2">
+                <a href={route('achievement.show', { achievement })} className="font-medium">
+                  {title}
+                  {game?.title ? ' ' : null}
+                </a>
+
+                {game?.title ? (
+                  <Trans
+                    i18nKey="<1>from</1> <2>{{gameTitle}}</2>"
+                    components={{
+                      1: <span />,
+                      2: <AchievementGameTitle game={game} />,
+                    }}
+                  />
+                ) : null}
+              </span>
+
+              {/* Points */}
+              {achievement.points && achievement.pointsWeighted ? (
```

Comment:
Event achievements are never showing points. We should probably show points (but not pointsWeighted) for event achievements worth more than 1 point.

Reply:
In latest, I've extracted the points display logic to a new child component: `<AchievementPoints />`.

This should now be satisfied by the following conditional within the component:
```tsx
if (!points || (isEvent && points === 1)) {
  return null;
}
```